src/main/java/org/folio/circulation/domain/UpdateItem.java

```diff
@@ -208,9 +209,13 @@ private ItemStatus itemStatusOnLoanUpdate(
     Loan loan,
     RequestQueue requestQueue) {
 
-    return loan.isClosed()
-      ? itemStatusOnCheckIn(requestQueue)
-      : CHECKED_OUT;
+    if(loan.isClosed()) {
+      return itemStatusOnCheckIn(requestQueue);
+    }
+    if(Objects.equals(loan.getItem().getStatus(), ItemStatus.DECLARED_LOST)) {
+      return loan.getItem().getStatus();
+    }
+    return  CHECKED_OUT;
```

Comment:
I really hate to do it, but there is an extra space here. Please remove it.

Reply:
Fixed.